nextflow_schema.json

```diff
@@ -630,5 +630,343 @@
         {
             "$ref": "#/definitions/institutional_config_options"
         }
-    ]
-}
+    ],
+    "properties": {
+        "enable_ms1": {
+            "type": "string",
+            "default": "true"
```

Comment:
Are you sure that these shouldn't all be `boolean` instead of strings?

If you really need them to be strings, then I guess you at least want to set them up with an `enum` (see the "advanced" option in the schema builder) to limit the different values that can be entered.